model/cloud.go

```diff
@@ -40,6 +40,14 @@ const (
 	SubscriptionFamilyOnPrem = SubscriptionFamily("on-prem")
 )
 
+type ProductSku string
+
+const (
+	ProductSkuStarter      = ProductSku("starter")
```

Comment:
I like adding these as constants. We have 3 sets of sku, those for cloud, those for on-prem, and "gov" skus (same plans of starter,professional,enterprise), but they have a different sku is all. Given that, I'd rather we store the full sku names and do a direct comparison above, rather than constructing it with `fmt.Sprintf`.  e.g.

api4/cloud.go
```go
	if product.SKU == model.SkuCloudStarter {
		w.Write(json)
		return
	}
```
model/cloud.go
```go
type ProductSku string

const (
	SkuStarterGov        = ProductSku("starter-gov")
	SkuProfessionalGov   = ProductSku("professional-gov")
	SkuEnterpriseGov     = ProductSku("enterprise-gov")
	SkuStarter           = ProductSku("starter")
	SkuProfessional      = ProductSku("professional")
	SkuEnterprise        = ProductSku("enterprise")
	SkuCloudStarter      = ProductSku("cloud-starter")
	SkuCloudProfessional = ProductSku("cloud-professional")
	SkuCloudEnterprise   = ProductSku("cloud-enterprise")
)
```

This makes it less awkward if self-hosted starts needing to do sku checks elsewhere in the code. My other issue with constructing the sku with product family name is that although all the cloud products have a cloud family and their skus start with `cloud-` currently, I still think of the subscription family and the sku prefix as two separate things that coincidentally happen to be the same.

Reply:
Makes sense. Definitely easier to read changes that use these SKUs if they have full names.